pkg/master/thirdparty/thirdparty.go

```diff
@@ -246,6 +246,9 @@ func (m *ThirdPartyResourceServer) InstallThirdPartyResource(rsrc *extensions.Th
 	if err != nil {
 		return err
 	}
+	if len(rsrc.Versions) == 0 {
```

Comment:
That makes sense, could you please leave a comment here? I would like to get this merged before this #36977